ddp-workspace/projects/ddp-testboston/src/assets/auth0-login-ar.js

```diff
@@ -0,0 +1,121 @@
+Auth0.registerLanguageDictionary("ar", {
```

Comment:
These two `auth0-login-ar.js` and `auth0-login.pt.js` files are needed because Auth0 does not have native support for Arabic or Portuguese, so we have to register our own dictionaries.